packages/ui/src/components/Button/ButtonLink.tsx

```diff
@@ -40,7 +40,7 @@ export function ButtonLink({
   focus:bg-accent-dark focus:border-transparent focus:ring-2 focus:ring-accent-lighter
   focus-within:bg-accent-dark focus-within:border-transparent focus-within:ring-2 focus-within:ring-accent-lighter
   active:bg-accent-dark active:border-accent-dark
-  disabled:bg-accent-lighter disabled:border-transparent
+  ${disabled && `bg-accent-lighter border-transparent pointer-events-none opacity-50`}
```

Comment:
No need to use template string quotes when template syntax is not used. Same for line 52.
```suggestion
  ${disabled && 'bg-accent-lighter border-transparent pointer-events-none opacity-50'}
```